packages/story-editor/src/components/footer/pagepreview/index.js

```diff
@@ -200,7 +192,7 @@ PagePreview.propTypes = {
   page: StoryPropTypes.page.isRequired,
   label: PropTypes.string,
   isCacheable: PropTypes.bool,
-  cachedImage: PropTypes.object,
+  cachedImage: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
```

Comment:
In which case it's an object and which case a string?

Reply:
am thinking blob for object and data-url for string

Reply:
Are you sure both are in use? I know it was giving a console warning about this but would be great to double check -- perhaps it's only using a `string` now? I'd assume it's either `null` or `string` -- could you double check before making this change?

Reply:
Okay will check this out

Reply:
Based on [Line 125](https://github.com/google/web-stories-wp/blob/b79def861f37c23051eb8edbc17dcea7f8f51722/packages/story-editor/src/components/footer/pagepreview/index.js#L125 ), this definition should be correct

Reply:
But based on this, it should always be an object, so something is still off there. Or in which case do you see it's set as a `string` in the code? Perhaps @barklund can chime in here since you worked on it last?

`setCachedImage({ pageId: page.id, cachedImage: null });` -> maybe the `cachedImage` from the `object` is passed to the `PagePreview` and not the whole object itself?

Reply:
Looks like `cachedImage` is used as a `string` inside the `PagePreview`, so it should probably still only be a string and not an object inside the `PagePreview`:
https://github.com/google/web-stories-wp/blob/b79def861f37c23051eb8edbc17dcea7f8f51722/packages/story-editor/src/components/footer/pagepreview/index.js#L177

Perhaps the naming inside there is confusing since the `cachedImage` is an object in one context and a `string` in another?

Reply:
OK, I'm checking now that yes -- `cachedImage` is passed as a `string` to `PagePreview` since the cached images are saved in the format of `{pageId: cachedImage, page2Id: cachedImage2, ...}` and `cachedImage` is a `string`.

So it should only be a `string` and not an `object`.

Reply:
I see what you mean. Done